README.md

```diff
@@ -185,45 +194,313 @@ interface LookerEmbedUser {
 
 ### Step 1 - Enable Embedding in your Looker instance
 
-(This is documented in more detail [here](https://docs.looker.com/r/sdk/sso-embed))
+Enabling SSO embedding is documented in more detail [here](https://docs.looker.com/r/sdk/sso-embed).
+Enabling cookieless embedding is documented in more detail [here](https://cloud.google.com/looker/docs/r/sdk/cookieless-embed).
+
+- Navigate to Admin > _Platform_ Embed on your Looker instance. This requires Admin privileges.
+- The demo server runs by default at [http://localhost:8080](http://localhost:8080). By adding that address to "Embedded Domain Whitelist" you can enabled the demo to receive messages from Looker.
+- Turn on "Embed SSO Authentication"
+- In order to use embedding you must generate an "Embed Secret" for SSO embedding and/or a JWT secret for cookieless embedding. Note that a Looker instance can support both types of embedding at the same time.
 
-* Navigate to Admin > *Platform* Embed on your Looker instance. This requires Admin privileges.
-* The demo server runs by default at [http://localhost:8080](http://localhost:8080). By adding that address to "Embedded Domain Whitelist" you can enabled the demo to receive messages from Looker.
-* Turn on "Embed Authentication"
-* In order to use embedding you must generate an "Embed Secret"
+Additional steps for cookieless embed:
+
+- Navigate to Admin > _Platform_ Embed on your Looker instance. This requires Admin privileges.
+- Generate an Embed JWT secret. This is used internally and the hosting application does not need to know what it is.
+- Navigate to Admin > _Labs_ Experimental on your Looker instance. This requires Admin privileges.
+- Toggle "Cookieless Embed" on. Note that a Looker instance can support SSO and Cookieless embed clients.
 
 ### Step 2 - Customize the Demo settings for your Looker instance
 
-* If you are using the main `demo.py`, provide your API credentials to the server by updating `demo/looker.ini` following [these instructions](https://community.looker.com/technical-tips-tricks-1021/the-how-to-on-initializing-the-sdk-with-different-profiles-in-your-ini-file-26846), with credentials obtained from [the Users page](https://docs.looker.com/reference/api-and-integration/api-auth).
+Note that `demo.py` and `demo_self_signed.py` have NOT been updated to support cookieless embedding. This section ONLY applies to SSO embedding. The cookieless embed demo currently requires the use of the development server.
+
+- If you are using the main `demo.py`, provide your API credentials to the server by updating `demo/looker.ini` following [these instructions](https://community.looker.com/technical-tips-tricks-1021/the-how-to-on-initializing-the-sdk-with-different-profiles-in-your-ini-file-26846), with credentials obtained from [the Users page](https://docs.looker.com/reference/api-and-integration/api-auth).
```

Comment:
@kenvlooker another link update (or two?)

Reply:
The community link is correct. The redirect from the docs.looker.com URL is in place, so that link is fine, too. That said, if we want to replace it with the current URL, that page is at https://cloud.google.com/looker/docs/api-auth.

Reply:
Updated